### ISSUE TITLE
APK request

### ISSUE THREAD

**Issue by discapacidad5** [mzdk100/rigela#2]
Hello, I would like to try this program. Could you please provide me with the APK file? Thank you.

**Comment by mzdk100** [comment]
Currently only supports the Windows platform, but will be able to support the Android platform in the future.

**Comment by discapacidad5** [comment]
> Currently only supports the Windows platform, but will be able to support the Android platform in the future.

Thank you for the information. I understand that the program currently only supports Windows. I would be happy to try the Windows installer. Could you please share it with me? I'm eager to see what this program can do.

**Comment by mzdk100** [comment]
You can click [here](https://github.com/mzdk100/rigela/blob/master/README-EN.md) to view README. Please refer to the instructions in the document to try the building steps. If you encounter any difficulties, please let me know.
Or if you want to directly obtain a binary file, I can also share it with you.

**Comment by discapacidad5** [comment]
> You can click [here](https://github.com/mzdk100/rigela/blob/master/README-EN.md) to view README. Please refer to the instructions in the document to try the building steps. If you encounter any difficulties, please let me know. Or if you want to directly obtain a binary file, I can also share it with you.

Hello,

Thank you for the information. I'm excited to try this program, but unfortunately, I am unable to compile the code myself due to my visual impairment.

Would it be possible for you to provide a pre-compiled binary for Windows that I can use directly? Your project aims to be accessible to people with disabilities, and providing pre-compiled binaries would make it more usable for individuals like myself.

Thank you for your understanding and assistance."

**Comment by mzdk100** [comment]
You can click [here](http://api.zhumang.vip:8080/rigela/rigela_x64/rigela-main.exe) Download the latest binary file of the build (main.exe).

**Comment by discapacidad5** [comment]
Thank you for providing the installer. Unfortunately, I'm still unable to use it due to the language barrier. The installer and the program itself are not in English. I have limited English proficiency and rely on translation tools to communicate.

If the program were in English, I could use it without any issues. However, the installer being in a different language poses a significant obstacle.

Would it be possible for you to provide an English version of the installer or the program? Additionally, I would be happy to assist with translating the program into Spanish if you provide me with the necessary translation files.

Thank you for your understanding and continued support.

**Comment by mzdk100** [comment]
The program defaults to using your system's language, but as you mentioned, if your system defaults to Spanish, the program may display an exception because we have not yet provided a translation for the Spanish language.
**If you can provide translation support, that would be great, you can check [here](https://github.com/mzdk100/rigela/blob/master/main/docs/development/I18N.md) Learn how to contribute to this repo.**

**Comment by mzdk100** [comment]
Of course, you can also change the language to English in the settings, so that you may be able to use it normally.

**Comment by discapacidad5** [comment]
"I would like to bring to your attention a significant issue that I encountered while trying to use the program. Despite my best efforts, I was unable to change the language to one that I understand. I tried various methods, but unfortunately, I was not successful.

The main problem is that the program does not offer an option to select the language at startup, which makes it difficult for non-native speakers to navigate the interface. I spent hours trying to find a way to change the language, but the instructions and options were all in a language that I do not understand.

I believe that it would be greatly beneficial to provide an option to select the language at startup, at least in English, which is a widely spoken language. This would allow users who do not speak the default language to access the program and use it effectively.

Another possible solution would be to implement a feature that detects the language of the user's computer and displays the program in that language. If the language of the computer is not supported, the program could default to English. This would make the program more accessible to a broader range of users.

Unfortunately, I was unable to use the program due to the language barrier, despite my interest in trying it out. I hope that you will consider my feedback and implement changes to make the program more user-friendly for non-native speakers. I would be eager to try the program again if I could access it in a language that I understand.

**Comment by discapacidad5** [comment]
"I apologize for the oversight. I had only read the most recent message and was not aware that there was a previous message. As a result, my previous response was actually addressing the last message. However, I see that you have resent the program, and I will attempt to work with this version to see if I can successfully use it. I will also carefully review the instructions and begin working on translating the program into Spanish.

**Comment by discapacidad5** [comment]
> El programa usa de manera predeterminada el idioma de su sistema, pero como mencionó, si su sistema usa el español de manera predeterminada, el programa puede mostrar una excepción porque aún no hemos proporcionado una traducción para el idioma español. **Si puede brindar asistencia para la traducción, sería genial. Puede consultar [aquí](https://github.com/mzdk100/rigela/blob/master/main/docs/development/I18N.md) Cómo contribuir a este repositorio.**

If the language is unavailable, such as Spanish in my case, it should default to English.

**Comment by mzdk100** [comment]
The idea of default to English is great, it can be implemented. Can you create a separate issue? I may start this job in the next few hours.

**Comment by mzdk100** [comment]
Hello, we have recently established an official website. Are you interested in experiencing it and providing improvement suggestions? You can download the latest binary files from the official website at any time in the future.
www.rigela.site

**Comment by wuwbobo2021** [comment]
Hello! Would you be interested in the Tactile project: <https://www.tactile-engineering.com/>, <https://conexusindiana.com/case-study/tactile-engineering-text/>, <https://github.com/tactile-eng>.

Well, it seems like Tactile hasn't made the graphics tablet design open-source, while mading some software dependencies public.
